_sass/pages/_who_uses_login.scss

```diff
@@ -0,0 +1,92 @@
+.container.who-uses-login {
+  .partners {
+    background: url(../img/who-uses-login/who-uses-illo-partners/who-uses-illo-partners.svg) left center no-repeat;
+    padding: 3rem 0 3rem 40%;
+  }
+
+  .security {
+    display: flex;
+    padding: 3rem 0;
+
+    > p {
+      flex-basis: 0;
+      flex-grow: 1;
+
+      &:first-child {
+        background: url(../img/who-uses-login/icon-shield/icon-shield.svg) left top no-repeat;
+        padding: 0 0 0 3rem;
+      }
+
+      &:last-child {
+        background: url(../img/homepage/icon-partner/icon-partners.svg) 0.4rem -.5rem no-repeat;
+        border-left: 1px solid darken($blue-lightest, 15%);
+        margin-left: 0.25rem;
+        padding: 0 0 0 4rem;
+      }
+    }
+  }
+
+  .mobile {
```

Comment:
Mentioned in an earlier comment, feels like we should be using USWDS helpers here (e.g. `at-media-max`) rather than our own custom implementation.

https://designsystem.digital.gov/design-tokens/spacing-units/#using-spacing-unit-tokens